src/components/10-atoms/checkbox/story.js

```diff
@@ -31,18 +36,24 @@ storyCheckbox.add(
 
     const wrapper = document.createElement('div');
     const template = html`
-      <axa-checkbox
-        refId="${refId}"
-        class="hover"
-        name="${name}"
-        variant="${variant}"
-        label="${label}"
-        ?disabled="${disabled}"
-        ?checked="${checked}"
-        ?required="${required}"
-        ?styled="${styled}"
-        error="${errortext ? 'Please accept our terms and conditions.' : ''}"
-      ></axa-checkbox>
+      <div
+        style="${variant && variant.indexOf('inverted') > -1
```

Comment:
Since we have the includes polyfilled, I would suggest to go with more modern code here, that is more readable:

`variant && variant.includes('inverted')`

But up to you. No biggie.

Reply:
No, "includes" is just for real arrays. A string is just implicit an array.

Reply:
no
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes

Reply:
Ah nice.... ok. Is that included in our polyfill for ie?